apps/wing/src/cli.ts

```diff
@@ -151,7 +150,7 @@ async function main() {
     .description(
       "Compiles a Wing program and runs all functions with the word 'test' or start with 'test:' in their resource identifiers"
     )
-    .argument("<entrypoint...>", "all entrypoints to test")
+    .argument("[entrypoint...]", "all files to test, globs are supported", ["{*main,*.test}.w"])
```

Comment:
We don't want to include things like sdfgsdfgmain.w

```suggestion
    .argument("[entrypoint...]", "all files to test, globs are supported", ["{main,*.main,*.test}.w"])
```

Reply:
According to @Chriscbr:
>It shouldn't matter if my tests are in `main.w` or `storage.test.w` or `storage.w` (you can't define tests in non-entrypoint files right now, but that's a bug at the moment).

Therefore I used `*.w` so that it accounts already for when the bug will be fixed.